Docs/Flavours.md

```diff
@@ -150,6 +150,15 @@ If providing just one file, name it `NVRAMTool.icns` if it is themed so that it
    - As another example of how flavours work: **ResetNVRAM:NVRAMTool** will look for `ResetNVRAM.icns`, then `NVRAMTool.icns` (and then, by OC default behaviour, `Tool.icns`, then `HardDrive.icns`)
    - _**NB**: Including **ResetNVRAM** anywhere in the flavour triggers picker audio-assist for "Reset NVRAM"_
 
+### SIP Tools
```

Comment:
In my opinion, this is an NVRAMTool. I.e. SIPTool:NVRAMTool. Thus the category.

Reply:
Good